webapp/redux/middleware/middleware.js

```diff
@@ -342,14 +342,14 @@ export default (store) => (next) => (action) => {
     case CREATE_NETWORK: {
       next(GeppettoActions.waitData('Instantiating the NetPyNE Model', GeppettoActions.layoutActions.SET_WIDGETS));
```

Comment:
@aranega this looks redundant (unless I am missing something) with the implementation you did, you will have this triggered first and then another waitData later with the wait message you implemented.

Reply:
@ddelpiano Thanks for the review! I wasn't sure I could remove it safely (that there was no side-effect somewhere), so I didn't want to disrupt that much the original code. I'll remove L343 in favour of the message display that is inside the `checkParametersThen` method.